pkg/ovalutil/pool.go

```diff
@@ -9,15 +9,20 @@ import (
 )
 
 var (
-	gzipPool sync.Pool
-	zstdPool sync.Pool
+	gzipPool = sync.Pool{
```

Comment:
just curious: what is the purpose of these pools? If multiple updaters update OVAL data, wouldn't there be a chance  something is accidentally overwritten?

Reply:
From the docs:
```
A Pool is safe for use by multiple goroutines simultaneously.

Pool's purpose is to cache allocated but unused items for later reuse, relieving pressure on the garbage collector. That is, it makes it easy to build efficient, thread-safe free lists. However, it is not suitable for all free lists. 
```
In this case an updater can grab a reader, do what it wants and return it to the pool, as long as the returning happens after the data is read there isn't a way to overwrite the data. Once the reader is returned it's reset and ready for the next client/updater/goroutine to use.